service/src/main/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsService.java

```diff
@@ -56,7 +69,26 @@ public AdditionalApplicationsBundle buildAdditionalApplicationsBundle(CaseData c
             .build();
     }
 
-    private C2DocumentBundle buildC2DocumentBundle(CaseData caseData, String uploadedBy, LocalDateTime uploadedTime) {
+    private String getSelectedApplicantName(Object applicantsList, String otherApplicant) {
+        DynamicList dynamicList = mapper.convertValue(applicantsList, DynamicList.class);
+
+        if (Objects.nonNull(dynamicList)) {
+            DynamicListElement selectedElement = dynamicList.getValue();
+
+            if (isNotEmpty(selectedElement)) {
+                if (APPLICANT_SOMEONE_ELSE.equals(selectedElement.getCode())) {
+                    return otherApplicant;
+                }
+                return selectedElement.getLabel();
+            }
+        }
+        return EMPTY;
```

Comment:
Would it be nicer to return an empty optional instead of an empty string here?

Reply:
updated to return Optional.empty()